src/main/scala/sbtprotoc/ProtocPlugin.scala

```diff
@@ -144,9 +144,74 @@ object ProtocPlugin extends AutoPlugin {
 
   override def projectConfigurations: Seq[Configuration] = Seq(ProtobufConfig)
 
-  def protobufGlobalSettings: Seq[Def.Setting[_]] =
```

Comment:
this was misnamed, but since I didn't see any reason for clients to use it, I renamed it and made the "new" def private to make the change verbosely breaking